### ISSUE TITLE
Intune App Wrapping 16.1.1 Error

### ISSUE THREAD

**Issue by krupapancholi22** [msintuneappsdk/intune-app-wrapping-tool-ios#90]
I have MacOS Big Sur and xcode 13, so I used Intune wrapping tool version 16.1.1 and it gives me following error when wrapping my ipa file. 

wrapping works ok with the previous wrapping tool but fails with 16.1.1 with following error.

Error: Failed to generate entitlements file.
Cause of the error to be provided for investigation:Error Domain=IntuneAppPackager Code=2 "/usr/bin/codesign exited with an error." UserInfo={NSLocalizedDescription=/usr/bin/codesign exited with an error.}

**Comment by Kyle-Reis** [comment]
Hi @krupapancholi22 - Are you able to upgrade to MacOS 12 (Monterey) or 13 (Ventura). Version 16.1.1 removed a dependency on a deprecated codesign argument and my suspicion is that the new argument it was replaced with is not supported on Big Sur

**Comment by krupapancholi22** [comment]
Hello Kyle-Reis, I upgraded MacOS to Ventura but that made me upgrade Xcode as well and after upgrading xcode, my app build doesn't work. So I have a different problem to deal with and different error to troubleshoot. Meanwhile, I picked up previously built app (the one on xcode13) and wrapped it using the tool 17.1.1. That finished the wrapping. There was some error with 16.1.1 but did not spend time in troubleshooting since anyway I was on Ventura, so just went with 17.1.1.

I have solution with Mac upgrade; but original problem basically was not resolved; I believe documentation says that for xcode 13, use version 16.1.1 and that is failing. So technically, this issue can be closed. But tool documentation should verify if xcode 13 and 16.1.1 are compatible or not.
